docs/installation/migrate-to-xcp-ng.md

```diff
@@ -4,37 +4,37 @@ sidebar_position: 3
 
 # Migrate to XCP-ng
 
-How to migrate from VMware, KVM, etc. to XCP-ng.
+Learn how to migrate from other virtualization platforms like VMware, KVM, and more to XCP-ng.
 
-This documentation will help you to make a migration to XCP-ng, from any most common other virtualization platform (VMware, KVM, etc.)
+This guide provides step-by-step instructions for migrating your virtual machines (VMs) from popular platforms to XCP-ng.
 
 :::note
-OVA import method will miss the information if the VM is running BIOS or UEFI mode. Double check your settings on your original system, and then enable (or not) UEFI on XCP-ng side for the destination VM. You can do so in VM advanced tab in Xen Orchestra.
+The OVA import method doesn't retain information about whether the VM uses BIOS or UEFI mode. Ensure you verify your VM's boot mode on the source platform and enable or disable UEFI accordingly on the XCP-ng side. You can adjust this setting in the **Advanced** tab of the VM configuration in Xen Orchestra.
 :::
 
 ## 🇽 From XenServer
 
-We got a dedicated section on [how to migrate from XenServer to XCP-ng](../../installation/upgrade#upgrade-from-xenserver).
+We provide a dedicated guide on [migrating from XenServer to XCP-ng](../../installation/upgrade#upgrade-from-xenserver).
 
 ## 🍋 From Citrix Hypervisor
 
-We got a dedicated section on [how to migrate from Citrix Hypervisor to XCP-ng](../../installation/upgrade#upgrade-from-xenserver).
+Follow our detailed instructions for [migrating from Citrix Hypervisor to XCP-ng](../../installation/upgrade#upgrade-from-xenserver).
 
 ## 🐼 From Xen on Linux
 
-If you are running Xen on your usual distro (Debian, Ubuntu…), you are using `xl` to manage your VMs, and also plain text configuration files. You can migrate to an existing XCP-ng host thanks to [the `xen2xcp` script](https://github.com/xcp-ng/xen2xcp).
+If you're running Xen on a Linux distribution (e.g., Debian, Ubuntu) using `xl` and plain text configuration files, you can migrate to an XCP-ng host using the [xen2xcp script](https://github.com/xcp-ng/xen2xcp).
 
-Check [the README](https://github.com/xcp-ng/xen2xcp/blob/master/README.md) for usage instructions.
+Refer to the [README](https://github.com/xcp-ng/xen2xcp/blob/master/README.md) for detailed usage instructions.
 
 ## 📦 From Virtualbox
 
-Export your VM in OVA format, and use Xen Orchestra to import it. If you have an issue on VM boot, check the [VMware](#-fromvmware) section.
+Export your VirtualBox VM as an OVA file, then use Xen Orchestra to import it. If you encounter boot issues, check the [VMware migration](#from-vmware) section for troubleshooting tips.
 
 ## 🇻 From VMware
 
-### XO V2V
+### XO V2V Migration
```

Comment:
Changing titles can break direct links to the section, either from within the documentation, or from blog posts, forum etc.